### ISSUE TITLE
Support for dark theme

### ISSUE THREAD

**Issue by tarjeieo** [Puzzlepart/did#54]
### tl;dr
Enable Dark Theme option

### Description
There should be a user setting for switching between dark and light theme

#### 8 pro tips for a successful dark mode theme:

- Pure black borders lines and box shadows 000.
- Dark black button backgrounds 222 with darker hovers 111.
- Soft black page background 333, three shades less than pure black.
- Gray for tab-focused outlines 777.
- Gentle white for text #CCC, three shades less than pure white.
- Bright white for headings #DDD or #EEE.
- Pure white for bold text #FFF.
- Colors with increased brightness and reduced saturation to prevent vibration.
